docs/install/containers.md

```diff
@@ -108,6 +107,12 @@ Bring everything up :rocket:
 docker-compose -f prom-stack.yml -f harvest-compose.yml up -d --remove-orphans
 ```
 
+### Note on Docker Logging Configuration
+
+By default, Docker uses the `json-file` logging driver which does not limit the size of the logs. This can lead to issues if your system runs out of space. Docker provides several options for logging configuration, including different logging drivers and options for log rotation.
```

Comment:
Maybe it is clearer to say 
"This can cause your system to run out of disk space."
instead of
"This can lead to issues if your system runs out of space."